### PR TITLE
[OPIK-539]: add a config tab and proxy;

### PR DESCRIPTION
**Pull request by aadereiko**
## Details
https://www.figma.com/design/4DchV5t7FXiwngzBUiqcZ3/Opik-UI?node-id=1584-639010&t=KpuRfR1MKIGoSTjV-0

### Moved the Feedback defintions to configuration tab and restyled it a bit:
Fixed the e2e tests for them
image

### Added AI Providers tab:
image
image
image
image

Removed the usage of tokens from local storage

### Playground improvements
Now, the playground uses the proxy to get the OpenAI calls. So, we don't need to put a token manually

Handling errors:
image

If a provider is set, the default model is picked:

image

## Resolves
https://comet-ml.atlassian.net/browse/OPIK-539

## Testing
Changed the tests for feedback definitions

### CODE REVIEW

File: apps/opik-frontend/src/components/pages/ConfigurationPage/AIProvidersTab/AIProvidersTab.tsx [hunk 1]
```diff
@@ -0,0 +1,153 @@
...
+  const [search, setSearch] = useState("");
+  const resetDialogKeyRef = useRef(0);
+  const [openDialog, setOpenDialog] = useState<boolean>(false);
+
+  const { data, isPending } = useProviderKeys(
+    {
+      workspaceName,
+    },
+    {
+      placeholderData: keepPreviousData,
```
Comment (ferc) [inline apps/opik-frontend/src/components/pages/ConfigurationPage/AIProvidersTab/AIProvidersTab.tsx:61]: We can remove later

File: apps/opik-frontend/src/components/ui/select.tsx [hunk 2]
```diff
@@ -109,23 +109,31 @@ const SelectLabel = React.forwardRef<
...
+  extends React.ComponentPropsWithoutRef<typeof SelectPrimitive.Item> {
+  withoutCheck?: boolean;
+}
+
 const SelectItem = React.forwardRef<
   React.ElementRef<typeof SelectPrimitive.Item>,
-  React.ComponentPropsWithoutRef<typeof SelectPrimitive.Item>
->(({ className, children, ...props }, ref) => (
+  SelectItemProps
+>(({ className, children, withoutCheck = false, ...props }, ref) => (
```
Comment (ferc) [inline apps/opik-frontend/src/components/ui/select.tsx:120]: @andriidudar take a look at it in case you have some strong opinion about it

**Review by ferc — APPROVED** [review]
Amazing code, looks very good and nice to have this feature ready!

File: apps/opik-frontend/src/components/ui/select.tsx (hunk 2 above)
Comment (andriidudar) [reply apps/opik-frontend/src/components/ui/select.tsx:120]: Good implementation, no comments from my side.

**Pull request merged**